src/gl.rs

```diff
@@ -435,7 +435,7 @@ pub trait Gl {
     fn get_program_binary(&self, program: GLuint) -> (Vec<u8>, GLenum);
     fn program_binary(&self, program: GLuint, format: GLenum, binary: &[u8]);
     fn program_parameter_i(&self, program: GLuint, pname: GLenum, value: GLint);
-    fn get_vertex_attrib_iv(&self, index: GLuint, pname: GLenum) -> GLint;
+    fn get_vertex_attrib_iv(&self, index: GLuint, pname: GLenum) -> Vec<GLint>;
```

Comment:
There are only two uses of this function on Servo side, so it will be trivial to fix the breakage there.